editor.planx.uk/src/@planx/components/FileUploadAndLabel/SelectMultipleFileTypes.tsx

```diff
@@ -163,7 +163,7 @@ export const SelectMultipleFileTypes = (props: SelectMultipleProps) => {
   };
 
   return (
-    <SelectMultiple<Option>
+    <SelectMultiple
```

Comment:
This type can be removed as it's redundant - it's already being inferred correctly from the `options` prop.